### ISSUE TITLE
Calling forward after the backward pass and parameter update

### ISSUE THREAD

**Issue by rohanpaleja27** [mjpyeon/pytorch-bayes-by-backprop#1]
Hi,

Thank you for keeping your code open source. While going through it, I see that you are calling a forward pass after your parameter update, which makes the printed training accuracy incorrect.

If there is a reason for this, please let me know as I am not highly familiar with BNNs.

Best,
